CONTRIBUTING.md

```diff
@@ -32,4 +32,6 @@ feat(api-explorer): add color variants
 
 ### Visual Regression Tests
 
-If you update the docs or the rendering changes, you'll need to update the snapshots. As most environments font configs are different, the simplest thing to do is grab the updated snapshots from github's actions. After a failed tests, the updated snapshots should be available for download.
+If you update the docs or the rendering changes, you'll need to update the snapshots. As most environments font configs are different, the simplest thing to do is grab the updated snapshots from the **Artifacts** section of your GitHub Actions workflow run (see [this failed workflow run](https://github.com/readmeio/markdown/actions/runs/1994189147) for an example).
+
+After a failed test, the updated snapshots should be available for download in the `image-snapshots` artifact. To update the snapshots, unzip the `image-snapshots` file and load its images into the `__tests__/browser/ci` directory. You can also view the (somewhat chaotic) image diffs in the `image-diffs` artifact.
```

Comment:
I really gave it my best to try to get these snapshots working in all environments, but 😞  https://github.com/readmeio/markdown/pull/395

Reply:
yeah I talked offline with Kelly about it a bit and I honestly think it makes the most sense to stick with GitHub CI for updating snapshots, at least for the time being

Reply:
What if we just committed the changes in the action? You can view the diffs in review step?

Reply:
Oops, just merged! I think I prefer the current approach, that way people aren't accidentally pushing bad snapshots (i.e. if a snapshot test is flaky) to the main branch.